pkg/controller/atlasproject/atlasproject_controller.go

```diff
@@ -160,6 +160,10 @@ func (r *AtlasProjectReconciler) Reconcile(context context.Context, req ctrl.Req
 					setCondition(ctx, status.PrivateEndpointReadyType, result)
 					return result.ReconcileResult(), nil
 				}
+				if result = DeleteAllNetworkPeers(context, projectID, ctx.Client.Peers, ctx.Log); !result.IsOk() {
+					setCondition(ctx, status.NetworkPeerReadyType, result)
```

Comment:
When deleting it also makes sense to unset the conditions for the peers 🤔

Reply:
What do you mean? When the project deleting, why is it makes sense to unset the conditions for the peers?

Reply:
Ah, true, no reason to do it here, I mixed the spot but it still makes sense to unset them somewhere)

Reply:
Sure. Please look at https://github.com/mongodb/mongodb-atlas-kubernetes/blob/dc943a1cf0ab10859525f6b2d067ee0e4522cdae/pkg/controller/atlasproject/network_peering.go#L43

Reply:
Great 👍